bower.json

```diff
@@ -22,7 +22,7 @@
   "homepage": "https://github.com/PolymerElements/paper-card",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0",
+    "polymer": "Polymer/polymer#^1.1.0",
```

Comment:
My logic was that ff someone has Polymer installed and they pull the new release of this element, bower will check whether any dependencies need to be updated, and update them. In this case, it will notice that there's a new Polymer release it cares about and pull the right one.

However, this new style thing doesn't work with 1.0.0, so maybe explicitly using `1.1.0` is better. Let me double check with @cdata tomorrow.